lib/mayaUsd/utils/dynamicAttribute.cpp

```diff
@@ -29,19 +29,27 @@ bool hasDynamicAttribute(const MFnDependencyNode& depNode, const MString& attrNa
     return depNode.hasAttribute(attrName);
 }
 
-MStatus createDynamicAttribute(MFnDependencyNode& depNode, const MString& attrName)
+MStatus createDynamicAttribute(
+    MFnDependencyNode&     depNode,
+    const MString&         attrName,
+    const DynamicAttrFlags flags)
 {
     MStatus status;
 
-    MFnTypedAttribute typedAttrFn;
-    MObject attr = typedAttrFn.create(attrName, "", MFnData::kString, MObject::kNullObj, &status);
+    MFnTypedAttribute attrFn;
+    MObject attr = attrFn.create(attrName, "", MFnData::kString, MObject::kNullObj, &status);
     CHECK_MSTATUS_AND_RETURN_IT(status);
 
-    typedAttrFn.setReadable(true);
-    typedAttrFn.setWritable(true);
-    typedAttrFn.setHidden(true);
-    typedAttrFn.setKeyable(false);
-    typedAttrFn.setStorable(true);
+    attrFn.setAffectsAppearance(isFlagSet(flags, DynamicAttrFlags::kAppearance) ? true : false);
```

Comment:
minor personal preference. We could remove all the ternary operators here since isFlagSet already returns the expected true/false

Reply:
Oh you're right, the code was a bit different before and that is a remnant